### PR TITLE
feat: add tether length offset property

### PR DESCRIPTION
**Pull request by seixasxbr**
Depends on:
- [ ] https://github.com/tidewise/bundles-rov/pull/248

# What is this PR for
Add a property to rov driver to facilitate offset removal
- [ ] [Shortcut](http://) 

# Results, How I tested
tested in live
unit test

# Checklist
- [x] Assign yourself  in the PR
- [x] Write unit tests (when relevant)

### CODE REVIEW

File: test/revolution_test.rb [hunk 1]
```diff
@@ -164,7 +163,7 @@ def raw_packet_input
                         },
                         "63B234C0A269": {
                             "acConnected": true,
-                            "distance": 100,
+                            "distance": 300,
```
Comment (kapeps) [inline test/revolution_test.rb:166]: why did you change it?

**Review by kapeps — APPROVED** [review]

File: test/revolution_test.rb (hunk 1 above)
Comment (seixasxbr) [reply test/revolution_test.rb:166]: No reason, I was just testing a different value. Could go back

**Review by DeboraTahara — APPROVED** [review]
